pkgs/development/python-modules/brother/default.nix

```diff
@@ -5,26 +5,28 @@
 , pytest-asyncio
 , pytest-error-for-skips
 , pytest-runner
-, pytest-tornasync
-, pytest-trio
 , pytestCheckHook
 , pythonOlder
 }:
 
 buildPythonPackage rec {
   pname = "brother";
-  version = "1.0.0";
+  version = "1.0.1";
   disabled = pythonOlder "3.8";
 
   src = fetchFromGitHub {
     owner = "bieniu";
     repo = pname;
     rev = version;
-    sha256 = "sha256-0NfqPlQiOkNhR+H55E9LE4dGa9R8vcSyPNbbIeiRJV8=";
+    sha256 = "sha256-Cfut6Y4Hln32g4V13xbOo5JdjPv2cH6FCDqvRRyijIA=";
   };
 
+  nativeBuildInputs = [
+    pytest-runner
+  ];
+
   postPatch = ''
-    substituteInPlace pytest.ini \
+    substituteInPlace setup.cfg \
       --replace "--cov --cov-report term-missing " ""
     substituteInPlace requirements-test.txt \
       --replace "pytest-cov" ""
```

Comment:
Uh, where is this requirements file used? I don't think it makes any sense :smile:

Reply:
`setup.py` seems to parse it -> https://github.com/bieniu/brother/blob/1.0.1/setup.py#L28

Reply:
But since we don't call `python setup.py test`, how would that get evaluated?

Reply:
It doesn't.